src/models/taskModel.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const taskSchema = new mongoose.Schema({
+    title: {
+        type: 'String',
+        required: [true, 'Please provide task title'],
+        description: {
+            type: 'String',
+            required: [true, 'Please provide task description'],
+        },
+    },
+    points: {
+        type: Number,
+        required: true
+    },
+    createdAt: Date,
+    assignedTo: {
+        type: mongoose.Schema.ObjectId,
+        ref: 'child'
+    },
+    completedAt: Date,
+    isApproved: Boolean  
```

Comment:
you can make the default value: false
so that when a new task is created, it is false..

Reply:
Thank you will add this